packages/react-form-renderer/src/files/form-renderer.js

```diff
@@ -46,21 +43,14 @@ const FormRenderer = ({
     return <SchemaErrorComponent name={schemaError.name} message={schemaError.message} />;
   }
 
-  const Template = FormTemplate
-    ? FormTemplate
-    : () => <div>{`FormRenderer is missing 'FormTemplate' prop: ({formFields, schema}) => <FormTemplate {...} />`}</div>;
```

Comment:
It's a required prop, so the warning will be there anyway.

We don't check other required props. (Should we check them?)

And it's probably better to cause a fail of the component, otherwise some test cases could pass.

Reply:
I think we can plan a future feature for prop types checking with links to docs for complex props like this. I think it would help more than the common prop type check.

Reply:
Yeah, let's do it in a complex way!